src/main/java/com/pmm/budgetcalculator/expensing/entity/mapper/ExpenseTypeMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.pmm.budgetcalculator.expensing.entity.mapper;
+
+import com.pmm.budgetcalculator.expensing.entity.ExpenseType;
+import com.pmm.budgetcalculator.expensing.entity.dto.ExpenseTypeDto;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface ExpenseTypeMapper {
+    ExpenseTypeDto entityToDto(ExpenseType expenseType);
+
+    ExpenseType dtoToEntity(ExpenseTypeDto expenseTypeDto);
+
+    List<ExpenseTypeDto> entityListToDtoList(List<ExpenseType> expenseTypeList);
```

Comment:
Mapping list to listDto is redundant. Iterating on list and mapping each element is acceptable and way to go